### PR TITLE
Add link to deployment and dc from pod page

### LINKED ISSUE #319: Pod page should link back to deployment and deployment config
There's no link to the RC or deployment that owns the pod on the pod page. This would be convenient for debugging. For instance, I was recently looking at an error in a pod log and needed to change an env var in the deployment config. It's a lot of clicks to get there.

### PR DESCRIPTION
**Pull request by spadgett**
If the pod is part of a deployment, add a link to the deployment and deployment config.

screen_shot_2016-12-14_at_3_14_20_pm

Closes #319
Replaces #1039

### CODE REVIEW

**Comment by spadgett** [comment]
For reference, here's the link on the replication controller page:

screen_shot_2016-12-14_at_3_28_22_pm

File: app/scripts/controllers/pod.js [hunk 1]
```diff
@@ -218,9 +218,13 @@ angular.module('openshiftConsole')
       $scope.containersEnv = containers;
     };
 
+    var annotation = $filter('annotation');
     var podResolved = function(pod, action) {
       $scope.loaded = true;
       $scope.pod = pod;
+      $scope.dcName = annotation(pod, 'deploymentConfig');
+      $scope.rcName = annotation(pod, 'deployment');
+      $scope.deploymentVersion = annotation(pod, 'deploymentVersion');
```
Comment (jwforres) [inline app/scripts/controllers/pod.js]: don't we want the same for pods that are part of other controllers? or are we waiting for the rebase to get pods with ownerRefs

Comment (spadgett) [inline app/scripts/controllers/pod.js]: > or are we waiting for the rebase to get pods with ownerRefs

Yes, waiting for owner refs. I don't see any way to do this for other controllers yet.

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
> There was an error talking to AWS.

[merge]

**Comment by jwforres** [comment]
[merge] AWS capacity issue

**Comment by spadgett** [comment]
[merge]

**Pull request merged**